geometry_msgs/Point zone_center
float32 zone_radius
---
bool success
string message

